adaptive_memory_v3.1.py

```
@@ -1495,16 +1495,16 @@ async def inlet(
                     bank_list_str = "\n".join([f"- {bank} {'(Default)' if bank == default_bank else ''}" for bank in allowed_banks])
                     response_msg = f"**Available Memory Banks:**\n{bank_list_str}"
                     await self._safe_emit(__event_emitter__, {"type": "info", "content": response_msg})
-                    body["messages"] = [] # Prevent LLM call
-                    body["prompt"] = "Command executed." # Placeholder for UI
+                    # Provide only user message to prevent empty content extraction in OpenWebUI's memory system
+                    # Don't include assistant message to avoid Gemini's "end with user role" requirement
+                    body["messages"] = [{"role": "user", "content": f"Command: {final_message}"}]
                     body["bypass_prompt_processing"] = True # Signal to skip further processing
                     return body
                 except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)